bitcoin-magazine/bitcoin-explained/explaining-bitcoin-addresses.md

```diff
@@ -0,0 +1,649 @@
+---
+title: "Explaining Bitcoin Addresses"
+transcript_by: mubarak23 via review.btctranscripts.com
+media: https://www.youtube.com/watch?v=R1kF1rnLvM8
+tags: ["script"]
+speakers: ["Sjors Provoost","Aaron Van Wirdum"]
+categories: ["podcast"]
+date: 2021-02-19
+---
+
+Aaron Van Wirdum: 00:00:00
+
+Before the show, a quick word from our sponsor.
+
+Sjors Provoost: 00:00:04
+
+What is up Bitcoiners? It's CK and I'm here to talk to you about the Bitcoin 2021 conference this June in sunny Miami, Florida. It is going to be June 4th and 5th and we are barreling towards a sold out conference. Already one third of our available tickets have been sold and way more than half of our whale passes have been sold. We are almost sold out completely on the whale pass. We have amazing speakers, Jack Dorsey, Chamath, Nick Szabo, Tony Hawk, Peter Todd, many, many more and many more to be announced. You can go to the website to check them all out. Mayor Suarez has welcomed us into the city with open arms and again, tickets are flying off the shelves. Seriously, I see the feed and like I've done many conferences. This one is is really has some something going for it. And honestly, guys, like, after all this after 2020, Bitcoiners want to grab a beer, hang out. We're bringing back Bitcoin 2019 vibes, times 10, prices times 10. Let's go. Use promo code Satoshi, all caps, SATOSHI for 10% off. You can go to the website, b.tc backslash conference. Again, that is b.tc backslash conference and get your ticket today. Today prices also are going up and hey we may just sell out so we have a hard cap can't really be flexible there don't wait
+
+Aaron Van Wirdum: 00:01:45
```

Comment:
I think we can start the transcript here

Reply:
Alright